pkg/sample/sampledeployment.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright (C) 2018 Synopsys, Inc.
+
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements. See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownershia. The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package sample
+
+import (
+	horizonapi "github.com/blackducksoftware/horizon/pkg/api"
+	"github.com/blackducksoftware/horizon/pkg/components"
+)
+
+// Returns a Horizon Deployment for the Sample
+func (specConfig *SampleSpecConfig) getSampleDeployment() *components.Deployment {
+	var replicas int32 = 1
+	deployment := components.NewDeployment(horizonapi.DeploymentConfig{
+		Name:      specConfig.config.Namespace,
+		Namespace: specConfig.config.Namespace,
+		Replicas:  &replicas,
+	})
+
+	deployment.AddPod(specConfig.getSamplePod())
+
+	deployment.AddMatchLabelsSelectors(map[string]string{
+		"app":  specConfig.config.Namespace,
+		"tier": specConfig.config.Namespace,
+	})
+
+	return deployment
+}
+
+// Returns a Horizon Pod for the Sample
+func (specConfig *SampleSpecConfig) getSamplePod() *components.Pod {
+	pod := components.NewPod(horizonapi.PodConfig{
+		Name: specConfig.config.Namespace,
+	})
+
+	pod.AddLabels(map[string]string{
+		"app":  specConfig.config.Namespace,
+		"tier": specConfig.config.Namespace,
+	})
+
+	pod.AddContainer(specConfig.getSampleContainer())
+
+	return pod
+}
+
+// Returns a Horizon Container for the Sample
+func (specConfig *SampleSpecConfig) getSampleContainer() *components.Container {
+	container := components.NewContainer(horizonapi.ContainerConfig{
+		Name:  specConfig.config.Namespace,
+		Image: "registry.hub.docker.com/vasiliys/public-test:latest",
```

Comment:
What's does this container do? Should we maybe use something more generic?

Reply:
This is an image Vasiliy made that repeatedly prints a message. I can change it to something more generic if you have something in mind. 
@VSharapov

Reply:
If that image isn't specifically needed then I'd use one from the common images in the community.  Like a redis or something else small.

Reply:
@msenmurugan suggested I change it to <<image_name>> and tell the user to change it if they want to in the docs.